File: fibonacci/fibonacci.py
# Fibonacci number is a series of numbers in which each number is the sum of the two preceding numbers.
# The first two numbers in the series are the number 1. Write a program to print first n Fibonacci Numbers.

# For example, one execution would look like this:

# Please enter a positive integer greater than 1: 4
# 1
# 1
# 2
# 3

n = int(input("Please enter a positive integer greater than 1: "))
n0 = 0
n1 = 1

print(n1)
for i in range(n - 1):
    n2 = n0 + n1
    print(n2)
    n0 = n1
    n1 = n2

File: fibonacci/fibonacci.cpp
#include <iostream>
using namespace std;

int main(int argc, char *argv[])
{
    int n;
    // n = atoi(argv[1]);
    cout << "Please enter a positive integer greater than 1: ";
    cin >> n;
    int n0 = 0;
    int n1 = 1;

    cout << n1 << endl;
    for (int i = 0; i < n - 1; i++)
    {
        int n2;
        n2 = n0 + n1;
        cout << n2 << endl;
        n0 = n1;
        n1 = n2;
    }

    return 0;
}